VideoApi/VideoApi/Mappings/ParticipantMapper.cs

```diff
@@ -0,0 +1,23 @@
+using VideoApi.Domain;
+
+namespace VideoApi.Mappings
+{
+    public static class ParticipantMapper
+    {
+        public static Participant MapParticipant(ParticipantBase participantBase)
+        {
+
+            var s = new Participant()
+            {
+                ParticipantRefId = participantBase.Id,
+                Name = participantBase.Name,
+                DisplayName = participantBase.DisplayName,
+                Username = participantBase.Username,
+                UserRole = participantBase.UserRole,
+                State = participantBase.State,
+            };
+            return s;
```

Comment:
return this in one line:

```suggestion
           return  new Participant()
            {
                ParticipantRefId = participantBase.Id,
                Name = participantBase.Name,
                DisplayName = participantBase.DisplayName,
                Username = participantBase.Username,
                UserRole = participantBase.UserRole,
                State = participantBase.State,
            };

```